src/compute/src/compute_state.rs

```diff
@@ -387,35 +423,46 @@ impl<'a, A: Allocate> ActiveComputeState<'a, A> {
     /// Send progress information to the coordinator.
     pub fn report_compute_frontiers(&mut self) {
         let mut new_uppers = Vec::new();
+        let mut new_frontier = Antichain::new();
```

Comment:
It seems we should be able to move `new_frontier` into the `for` loop below? It might save a `clone` call. But, I might be missing something, so ignore me if that's so!

Reply:
Yes, good catch!